angr/analyses/cfg/cfg_accurate.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, *args, **kwargs):
 
         self.cfg_node = None
         self.sim_successors = None
-        self.error_occurred = None
+        self.exception_info = None
```

Comment:
Adding a little documentation of what `exception_info` is storing will be awesome.